packages/driver/cypress/integration/commands/assertions_spec.js

```diff
@@ -127,6 +127,17 @@ describe('src/cy/commands/assertions', () => {
       }).should('deep.eq', { foo: 'baz' })
     })
 
+    // https://github.com/cypress-io/cypress/issues/16006
+    it(`does not show more than one .should('contain') assertion when chained after .should('be.visible')`, function () {
```

Comment:
I feel like this is not accurate to what was happening. The problem was that it would show multiple 'visible' assertions when there was only 1 visible assertion and only show one of the contains assertions even though there were 2 contains assertions. 
```suggestion
    it(`shows all .should('contain') assertions when chained after .should('be.visible')`, function () {
```